### ISSUE TITLE
dispyscheduler crashes with error "AttributeError: '_Scheduler' object has no attribute 'scheduler_task'"

### ISSUE THREAD

**Issue by dismine** [pgiri/dispy#100]
Hi,
I run my scheduler in daemon mode. Looks like here is error because it crashes with traceback:

```
Traceback (most recent call last):
  File "/home/user/project/env/bin/dispyscheduler.py", line 2173, in <module>
    scheduler.scheduler_task.value()
AttributeError: '_Scheduler' object has no attribute 'scheduler_task'
```

**Comment by altaetran** [comment]
Having this problem as well. Can anyone comment on this?

**Comment by pgiri** [comment]
It is already fixed. Try latest release 4.8.5.

**Comment by altaetran** [comment]
I get this error whenever I try to run dispyscheduler.py as a background process using terminal (& process). I suspect it might be interpreting the & strangely, so I just used a screen process and it worked fine. Thank you for your prompt feedback!

**Comment by pgiri** [comment]
Yes, it was a problem with starting in daemon mode (background process). See the [fix](https://github.com/pgiri/dispy/commit/285fbc15c4b47217aa32a548d859e71c0bd16af9)
